### PR TITLE
feat(growth): CTA Performance Upsell in Issue Alert Email

### PR DESCRIPTION
**Pull request by PixelDust22**
image

The banner is not displayed on mobile.
image

### CODE REVIEW

File: src/sentry/templates/sentry/emails/error.html [hunk 1]
```diff
@@ -38,7 +38,21 @@
   </div>
 {% endblock %}
 
-{% block main %}
+{% block content %}
+{% if group.project.flags.has_transactions %}
```
Comment (scefali) [inline src/sentry/templates/sentry/emails/error.html:42]: @Neo-Zhixing We only want to show this block if the project doesn't have transactions, this looks like the opposite of that

**Review by scefali — COMMENTED** [review]
@Neo-Zhixing can you post a screenshot of what this looks like on a mobile view as well?

File: src/sentry/templates/sentry/emails/error.html (hunk 1 above)
Comment (Neo-Zhixing) [reply src/sentry/templates/sentry/emails/error.html]: Great catch! Thanks!

**Review by scefali — COMMENTED** [review]
@Neo-Zhixing doesn't look great on mobile…maybe we should just hide it on that view if possible like we do for the Slack button. Let’s talk about it today during our meeting.

**Review by scefali — APPROVED** [review]

**Pull request merged**